workshop_app/templates/workshop_app/propose_workshop.html

```diff
@@ -11,11 +11,14 @@
     <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
 
     <script>
+    var dateToday = new Date();
+    dateToday.setDate(dateToday.getDate() + 2);
```

Comment:
setDate() automatically handles it! 
i.e next month and it's respective date will be considered.